### ISSUE TITLE
opencv python gui window receiving last keypress from previous scripts even after closing wsl

### ISSUE THREAD

**Issue by ta946** [microsoft/wslg#213]
# Environment

```
Windows build number: 10.0.21376.1
Your Distribution version: Ubuntu 16.04
Your WSLg version: Windows Subsytem for Linux WSLg Preview 1.0.19
python 3.7.7
opencv-contrib-python  4.2.0.34
no x11 server, $DISPLAY=:0
```

# Steps to reproduce

create a python file, test.py then run the file
```
import cv2
cv2.namedWindow('img')
k = cv2.waitKey(0)
print(k, chr(k))
```
python test.py

A window called img will appear, focus the window then press any key, eg: "F". the window will close and print the key that was pressed.

Rerun the python script, the window will automatically close and print the previous key pressed

#  Expected behavior

Rerunning the python script should show the "img" window and wait for a keypress before closing and printing the pressed key

# Actual behavior

The python script immediately closes without any key pressed and prints the value of the key pressed the last time the script was run

cv2.destroyAllWindows() before closing makes no difference. Closing the wsl terminal also makes no difference, only after restarting wsl using --shutdown does it run as normal on the first run, then again does the same thing.

It reminds me of some glitches of using an x11 server that go away when you restart the x11 server, so I'm guessing it has to do with wayland and restarting wayland would have the same effect as restarting wsl

**Comment by hideyukn88** [comment]
@ta946 , thanks for reporting the issue, this sounds same issue as #207.

**Comment by spronovo** [comment]
Thanks for the report. Closing as a duplicate of #58 that we're using to track this issue.

**Comment by Alkaidcc** [comment]
same issue
any solution to fix it?

**Comment by hideyukn88** [comment]
@Alkaidcc, have you tried the latest WSL/WSLg from aka.ms/wslstorepage?

**Comment by Alkaidcc** [comment]
> @Alkaidcc, have you tried the latest WSL/WSLg from aka.ms/wslstorepage?

After installing the latest WSL/WSLg from aka.ms/wslstorepage, the problem solved. Thanks for your reply!
